rails-blog/app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
-  layout 'mailer'
+  default from: "from@example.com"
```

Comment:
Our rubocop uses single quotes, can you please configure your formatting to use the toplevel rubocop config?

Reply:
This repository does not contain a Rubocop config. I've updated this file to how Rails generates the application.

Reply:
Ah yeah... okay. We can do that later if we still want to.